04_setup_ironic.sh

```diff
@@ -75,6 +55,12 @@ sudo podman run -d --net host --privileged --name ironic --pod ironic-pod \
      --env MARIADB_PASSWORD=$mariadb_password \
      -v $IRONIC_DATA_DIR:/shared ${IRONIC_IMAGE}
 
+sudo podman run -d --net host --privileged --name ipa-downloader --pod ironic-pod \
+     -v $IRONIC_DATA_DIR:/shared ${IPA_DOWNLOADER_IMAGE} /usr/local/bin/get-resource.sh
```

Comment:
Do these actually need to be privileged?  I forget why we needed that, but it'd be good to ensure only those containers that really need it have it specified?

Similarly with host networking, I guess we need that for the containers which run a server, but probably not for those that just download and write to a shared volume?

I tested the containers in a pod via `podman play` on my host without either and it worked fine (although podman play on the RHCOS image seems to segfault, so we'll have to stick to the manual script approach for starting the static-pod on the bootstrap node).